tests/providers/amazon/aws/hooks/test_glue_crawler.py

```diff
@@ -160,6 +160,18 @@ def test_remove_all_tags(self, mock_get_conn):
             ResourceArn=self.crawler_arn, TagsToRemove=["test", "bar"]
         )
 
+    @mock_sts
+    @mock.patch.object(GlueCrawlerHook, "get_conn")
+    def test_update_missing_tags(self, mock_get_conn):
+        mock_config_missing_tags = deepcopy(mock_config)
+        mock_config_missing_tags.pop("Tags")
+        mock_get_conn.return_value.get_crawler.return_value = {"Crawler": mock_config_missing_tags}
+
+        assert self.hook.update_crawler(**mock_config_missing_tags) is False
```

Comment:
Should we also add a testcase for when `Tags` key is in config?

Reply:
i think `test_replace_tag` function is testing the above case.

Reply:
yeah. Can you double check it @utkarsharma2 ?

Reply:
Ya, they are covered. I got confused by these lines, it seems like they don't add any value to the test. Test seems to work even after removing them. 
https://github.com/apache/airflow/blob/d01248382fe45a5f5a7fdeed4082a80c5f814ad8/tests/providers/amazon/aws/hooks/test_glue_crawler.py#L169-L170

Reply:
I removed those two lines